package.json

```diff
@@ -84,12 +84,12 @@
     "vscode-proxy-agent": "^0.12.0",
     "vscode-regexpp": "^3.1.0",
     "vscode-textmate": "7.0.1",
-    "xterm": "4.19.0-beta.29",
-    "xterm-addon-search": "0.9.0-beta.26",
+    "xterm": "4.19.0-beta.41",
```

Comment:
@meganrogge this probably includes fixes for other vscode issues that should be closed.